src/function/init.js

```diff
@@ -73,7 +73,7 @@ function($, emojione, blankImg, slice, css_class, emojioneSupportMode, invisible
                         options.search ?
                         function() {
                             self.search = $("<input/>", {
-                                "placeholder": "SEARCH",
+                                "placeholder": options.searchPlaceholder || "",
```

Comment:
`"placeholder": options.searchPlaceholder` would be enough here no? as options.searchPlaceholder would always be set

Reply:
@pederjohnsen You're right, but I try to code defensively. If someone accidentally passes a null var as the placeholder, at least this way it wont explode :-)